dart/async/lib/src/interfaces.dart

```diff
@@ -57,30 +57,83 @@ abstract class PageLoaderMouse {
       {PageLoaderElement eventTarget, bool sync: true});
 }
 
+/// Base class for anything in the DOM that users can interact with via
+/// pageloader (e.g. HTML Nodes, the document itself, etc).
 abstract class PageLoaderElement {
+  /// The [PageLoader] instance associated with this element.
   PageLoader get loader;
 
+  /// The shadow root hosted by this element.
   Future<PageLoaderElement> get shadowRoot;
+
+  /// The text content of this element.
+  ///
+  /// This should return the value of <element>.innerText (see
```

Comment:
I don't think this will render well in Dartdocs. Maybe:

`<element>.innerText`